$NetBSD: patch-Makefile,v 1.2 2023/02/13 23:38:48 thorpej Exp $

- Install the man page into the correct location for pkgsrc.
- Fix the version number.

--- Makefile.orig	2022-09-11 04:44:06.000000000 +0000
+++ Makefile	2023-02-13 22:30:20.142674405 +0000
@@ -24,7 +24,7 @@ MANDIR ?= $(PREFIX)/share/man
 ##########################################################################
 
 NAME = minipro
-VERSION = 0.5
+VERSION = 0.6
 
 # If we're working from git, we have access to proper variables. If
 # not, make it clear that we're working from a release.
@@ -72,7 +72,7 @@ BIN_INSTDIR=$(DESTDIR)$(PREFIX)/bin
 LIB_INSTDIR=$(DESTDIR)$(PREFIX)/lib
 SHARE_INSTDIR=$(DESTDIR)$(PREFIX)/share/minipro
 INCLUDE_INSTDIR=$(DESTDIR)$(PREFIX)/include/libminipro
-MAN_INSTDIR=$(DESTDIR)$(PREFIX)/share/man/man1
+MAN_INSTDIR=$(DESTDIR)$(PREFIX)/man/man1
 
 UDEV_DIR=$(shell $(PKG_CONFIG) --define-variable=prefix=$(PREFIX) --silence-errors --variable=udevdir udev)
 UDEV_RULES_INSTDIR=$(DESTDIR)$(UDEV_DIR)/rules.d
